clients/lsp-lua.el

```diff
@@ -556,6 +556,87 @@ and `../lib` ,exclude `../lib/temp`.
   :server-id 'lua-language-server
   :download-server-fn #'lsp-lua-language-server-install))
 
+;;; lua-language-server-latest
+
+(defgroup lsp-lua-language-server-latest nil
+  "Lua LSP client, provided by the Lua Language Server (GitHub release)."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/sumneko/lua-language-server"))
+
+(defcustom lsp-clients-lua-language-server-latest-bin
+  (f-join lsp-clients-lua-language-server-install-dir
+          "bin/"
+          (pcase system-type
+            ('gnu/linux "lua-language-server")
+            ('darwin "lua-language-server")
+            ('windows-nt "lua-language-server.exe")
+            (_ "lua-language-server")))
+  "Location of Lua Language Server (latest)."
+  :group 'lsp-lua-language-server-latest
+  :risky t
+  :type 'file)
+
+(defcustom lsp-clients-lua-language-server-latest-main-location
+  (f-join lsp-clients-lua-language-server-install-dir
+          "main.lua")
+  "Location of Lua Language Server main.lua (latest)."
+  :group 'lsp-lua-language-server-latest
+  :risky t
+  :type 'file)
+
+(defcustom lsp-clients-lua-language-server-latest-args '("-E")
+  "Arguments to run the Lua Language server (latest)."
+  :group 'lsp-lua-language-server-latest
+  :risky t
+  :type '(repeat string))
+
+(defun lsp--find-latest-lua-ls-release-url ()
```

Comment:
can you change all variables and functions to be prefixed with `lsp-lua-`. All that are not prefixed like that are due to the fact that initially all clients were in single file `lsp-clients`